apps/meteor/server/lib/channelExport.ts

```diff
@@ -53,7 +53,7 @@ type ISentViaEmail = {
 };
 
 export const sendViaEmail = (data: ExportEmail, user: IUser): ISentViaEmail => {
-	const emails = data.toEmails.map((email) => email.trim()).filter(Boolean);
+	const emails = data.toEmails.map((email) => email.trim()).filter(Boolean) || [];
```

Comment:
is there any case where it might fall into the `or`
```suggestion
	const emails = data.toEmails.map((email) => email.trim()).filter(Boolean);
```